extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -59,12 +59,13 @@ export const StateProperty = {
   AD_STATE: 'adstate',
   BOOKEND_STATE: 'bookendstate',
   DESKTOP_STATE: 'desktopstate',
-  HAS_AUDIO_STATE: 'hasaudiostate',
+  STORY_HAS_AUDIO_STATE: 'hasaudiostate',
```

Comment:
nit: I would call this `storyaudiostate`. Just to be consistent with `pageaudiostate`

Reply:
Also, we need to add these new states to the `State` `typedef` above in this file.